### PR TITLE
fix path update error with openenclaverc

### PR DESCRIPTION
**Pull request by mikbras**
This PR fixes the path update error reported by issue #1742.

### CODE REVIEW

File: cmake/openenclaverc.in [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+
+    # Save this variable so it can be removed when path is extended again.
+    export __OPEN_ENCLAVE_CURRENT_BINDIR=$1
+
+    # Add ${__OPEN_ENCLAVE_CURRENT_BINDIR} to the path:
+    export PATH=${PATH}:${__OPEN_ENCLAVE_CURRENT_BINDIR}
+}
+
 # Update PKG_CONFIG_PATH.
 export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_DATADIR@/pkgconfig
```
Comment (radhikaj) [inline cmake/openenclaverc.in]: Can we make this so this directory if put before ${PKG_CONFIG_PATH}
So export PKG_CONFIG_PATH=CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_DATADIR@/pkgconfig:${PKG_CONFIG_PATH}

**Comment by andschwa** [comment]
I wouldn't merge this until PKG_CONFIG_PATH is fixed too.

**Comment by kuanggang-git** [comment]
Can one of the admins verify this patch?

**Comment by achamayou** [comment]
@kuanggang-git there's a long discussion under #1742 about this and whether there should be an rc file at all: https://github.com/openenclave/openenclave/issues/1742

A number of folks there have suggested the rc file is probably more trouble than it's worth, and seeing how it's had this known issue open for now 10 months, I have to agree.

@radhikaj @mikbras you both seemed to be in favour of keeping the rc file, could you please take another look and help decide if this should be fixed or removed?

**Comment by mingweishih** [comment]
Closing this PR for now, feel free to reopen when needed.

**Pull request closed without merging**